suite-native/app/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@suite-native/app",
     "version": "1.0.0",
-    "suiteNativeVersion": "24.3.1",
+    "suiteNativeVersion": "24.4.0",
```

Comment:
Nice, it seems to app version matches the weather outside because in Olomouc there is already April out there :D

Reply:
This means we're developing for april release